zfsbootmenu/lib/zfsbootmenu-core.sh

```diff
@@ -1950,7 +1952,17 @@ zreport() {
   colorize orange "\n>> ZFSBootMenu commandline\n"
   get_zbm_kcl | kcl_assemble
 
-  colorize orange "\n\n>> ZFS/SPL module information\n"
+  colorize orange "\n\n>> Enabled hooks\n"
```

Comment:
i think the first `\n` should be on its own line grouped with the previous stanza just so we don't have to keep moving it around if things get added

Reply:
shifted it around slightly so that the line return is logically tied to `kcl_assemble`